Makefile

```diff
@@ -20,7 +20,7 @@ lint:
 
 .PHONY: mypy
 mypy:
-	mypy peakina
+	mypy .
```

Comment:
I think it's ok to enforce mypy only on the source code and not the tests. Why this change?

Reply:
I find reassuring to know that my tests respect the type annotations, + in some situations it can help to spot wrong type annotations.